src/dmclock/sim/src/ConfUtils.cc

```diff
@@ -38,8 +38,8 @@ using std::string;
 
 ////////////////////////////// ConfLine //////////////////////////////
 ConfLine::
-ConfLine(const std::string &key_, const std::string val_,
-      const std::string newsection_, const std::string comment_, int line_no_)
+ConfLine(const std::string &key_, const std::string &val_,
+      const std::string &newsection_, const std::string &comment_, int line_no_)
```

Comment:
it's my understanding that changes to the dmclock subtree need to go through https://github.com/ceph/dmclock

Reply:
I see. I'll drop this change and open a pull request on the dmclock repository instead then.